cluster/gce/windows/common.psm1

```diff
@@ -541,5 +542,19 @@ function Test-IsTestCluster {
   return $false
 }
 
+# Returns true if this node uses a TPM to bootstrap its authentication to the
+# master. $KubeEnv is a hash table containing the kube-env metadata
+# keys+values.
+function Test-NodeUsesTpmBootstrap {
+  param (
+    [parameter(Mandatory=$true)] [hashtable]$KubeEnv
+  )
+
+  if ($KubeEnv.Contains('TPM_BOOTSTRAP_CERT')) {
```

Comment:
I don't know this language, but can this be:
`return $KubeEnv.Contains('TPM_BOOTSTRAP_CERT')` ?

Reply:
Done.

Do you think this is the best way to test if this node should use the auth plugin, or is there a way that's more reliable / less likely to change in the future?

Reply:
That's roughly what we do for linux https://github.com/kubernetes/kubernetes/blob/8af6906d1f300ff739ac1a846d4108d134412103/cluster/gce/gci/configure.sh#L289-L291